# Configuration file for webcam video capture
is_high_memory_usage_topic: "data_capture/is_memory_usage_exceeded"
webcam_capture:
  cam_settings:
    device_index: 3
    frames_per_second: 30.0
    video_type: "mp4"
    video_dimensions: "480p"
